### PR TITLE
add colored graph of 7-day incidence rate

### PR DESCRIPTION
**Pull request by kirienko**
We should have done it a long time ago...
image

I disabled `plot_time_step` from showing in the `overview` function and inserted 7-day incidence rate _instead_, which is much more demanded now.

I also invested some time into the colormap: the line is green on the bottom, then it's yellow around 30, then in becomes more red, and above 100 it's totally maroon:
image

### CODE REVIEW

**Comment by kirienko** [comment]
Okay, I'm done with the design. Will write a test tomorrow.

**Comment by kirienko** [comment]
Look like this now:

image

File: oscovida/oscovida.py [hunk 1]
```diff
@@ -742,6 +744,47 @@ def plot_time_step(ax, series, style="-", labels=None, logscale=True):
     return ax
 
 
+def plot_incidence_rate(ax, cases: pd.Series, country: str = None, region: str = None, subregion: str = None,
+                        dates: str = None, weeks: int = 0):
```
Comment (RobertRosca) [inline oscovida/oscovida.py:748]:
```suggestion
def plot_incidence_rate(ax, cases: pd.Series, country: str = None,
                        region: str = None, subregion: str = None,
                        dates: str = None, weeks: int = 0):
```

Line's a bit long

Comment (RobertRosca) [reply oscovida/oscovida.py:748]: Eh doesn't really matter actually

**Comment by RobertRosca** [comment]
I like it! Probably on your list already, but the plot on the front page is the one created by the `save_frontpage_figure` function in the `tools/generate-individiual-plots.ipynb` notebook, so that might(?) have to be updated to use the new plotting style as well.

**Comment by fangohr** [comment]
Very nice - thank you!

A word of warning on the the colour of the line: it looks cool, but implies that values below 25 are okay (because it is green). That's not necessarily what we want to say - some people push for a zero COVID strategy. On the other hand, I think the colour could be useful to be able to compare curves from different regions that you cannot see at the same time. So I am happy to go with this and try it.

Should this replace the plot with the daily new infections? I would argue that the new plot should be there _in addition_. We have quite a scientific user base (I imagine), and they probably want to see, for example, how the reported numbers change from day to day, even if the weekly average is more meaningful.

Also: we must update the documentation at https://oscovida.github.io/plots.html once we have decided.

**Comment by fangohr** [comment]
We could consider dropping the current plot 1 (in terminology of https://oscovida.github.io/plots.html). That was interesting in the beginning, but now doesn't provide much data. Would you agree? This could make space for the new incidence plot.

**Comment by kirienko** [comment]
> We could consider dropping the current plot 1 (in terminology of https://oscovida.github.io/plots.html). That was interesting in the beginning, but now doesn't provide much data. Would you agree? This could make space for the new incidence plot.

Yeah, that's exactly what I've done already :smiley:

File: oscovida/oscovida.py (hunk 1 above)
Comment (kirienko) [reply oscovida/oscovida.py]: Done, thanks!

**Comment by kirienko** [comment]
Okay, seeing no objections, I merge this. Documentation updates will follow in a separate branch. I will also try to update the front page as @RobertRosca suggested.

**Pull request merged**